src/pages/Inquiry/index.tsx

```diff
@@ -1,49 +1,36 @@
 import { ReactComponent as Write } from 'assets/svg/inquiry/write.svg';
-import { ReactComponent as Move } from 'assets/svg/inquiry/myinquiry.svg';
 import usePostList from 'pages/Post/hooks/usePostList';
 import { useState } from 'react';
-import { Link } from 'react-router-dom';
 import styles from './Inquiry.module.scss';
-import Pagination from './pagination';
-import Datatable from './datatable';
+import Pagination from '../../components/Pagination';
+import Datatable from '../../components/DataTable';
```

Comment:
절대 경로 지정 룰이 없었나보네요?
다른 프로젝트들처럼 린트 룰도 추가하는건 어떨까요?
```suggestion
import Pagination from 'components/Pagination';
import Datatable from 'components/DataTable';
```

Reply:
수정햇습니다!
ce914a339b8c844c0deecff9860375ddd13372f9

Reply:
> 다른 프로젝트들처럼 린트 룰도 추가하는건 어떨까요?

이거에 대해선 어떻게 생각하시나요?? 🧐

Reply:
1737fcf3d126eeefb9efc5f3aca6578713f57f21 
코인 오너 프로젝트를 참고해서 린트 룰 추가했습니다!